hail/src/main/scala/is/hail/expr/types/physical/PCanonicalLocus.scala

```diff
@@ -35,13 +35,17 @@ final case class PCanonicalLocus(rgBc: BroadcastRG, required: Boolean = false) e
 
   val representation: PStruct = PCanonicalLocus.representation(required)
 
-  def contig(region: Code[Region], off: Code[Long]): Code[Long] = representation.loadField(region, off, 0)
+  def contig(off: Code[Long]): Code[Long] =
```

Comment:
why the newline in these methods? I think it's no more legible, generally we only do this if the line is too long (>120)